CMakeLists.txt

```diff
@@ -41,6 +39,7 @@ add_library(roboteam_utils STATIC
         include/roboteam_utils/json.hpp
         #include/roboteam_utils/BiMap.h
         #include/roboteam_utils/Graph.h
+        include/roboteam_utils/Definitions.h
         include/roboteam_utils/Statistics.h
         )
 
```

Comment:
Why are headers in add_library, they shouldn't get compiled.

Reply:
1. For consistency
2. Headers can contain definitions: if we ever want to switch between static/dynamic libraries then we would need to include them again.
3. It's clearer what code is used in what target if something only exists as header.

Reply:
1. but why were they there in the first place
2. Well if headers contain definitions then you include them into a .cpp file and then the translation unit has them.
3. not sure what you mean